src/interp/interp.h

```diff
@@ -506,12 +506,20 @@ class Store {
   template <typename T>
   friend class RefPtr;
 
+  struct GCContext {
+    int call_depth;
+    std::vector<bool> marks;
+    std::vector<size_t> untraced_objects;
```

Comment:
Just my suggestion.
It seems that these two vectors are allocated for calculation in GC(`Collect`) only, and never reclaimed.
If you concern about memory consumption, how about release these two vectors after `Collect`  calling like `shrink_to_fit()` vector function?

Reply:
But if collection happens frequently is there is advantage to grown in a shrinking these between collections?  Wouldn't that just add to fragmentation and cause the allocator do do extra work?

Reply:
I think that it depends on the situation.
If you put a high priority on memory usage, shrinking after GC would be a better choice.

To relieve fragmentation in shrinking, we can consider another option which just releases the whole space allocated for these vectors like below.
```c++
std::vector<bool>().swap(marks); // convert `marks` to an empty vector
```

AKAIK, `Collect` is not called inside wabt engine. So, user should call it explicitly to reclaim unused memory.
It would be great if GC is triggered automatically inside wabt engine (future work?)

Reply:
I have also wondering when calling GC automatically would be appropriate. It seems majority of GC objects are created by a user before validation, and should be released after a byte code is released. It seems the engine creates the trap object, but that should be very rare. Is there any object type which is created frequently by the engine? Maybe the frame object?